ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+<!--
+
+Hello! 
+Please take a quick look at those things down there. They're quite important.
+Really! We wrote them for you. Yes, you! With utmost care. Read them.
+If you're filing a bug, please include every step so as to help us reproduce it on our machines. If you're unsure about how to file an issue, use the issue template. If you need any help regarding usage of coala, check out the documentation or hit us up on chat. You can ignore or delete this text, it is commented and won't appear when the issue is submitted or previewed.
+
+Chat: https://coala.io/chat
+Documentation: https://docs.coala.io
+
+-->
+
+Filing Issues
+-------------
+
+When filing an issue, please use this template:
+
+::
```

Comment:
Please note the difference between markdown and reStructuredText syntax. You don't need `::` in markdown file.

Reply:
Oh, sorry. Would resolve the same in the next commit.